Index: web/ControleurFrontale.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n/*\r\nrequire_once __DIR__ . '/../src/Lib/Psr4AutoloaderClass.php';\r\n// initialisation\r\n$loader = new App\\Covoiturage\\Lib\\Psr4AutoloaderClass();\r\n$loader->register();\r\n// enregistrement d'une association \"espace de nom\" → \"dossier\"\r\n$loader->addNamespace('App\\Covoiturage', __DIR__ . '/../src');\r\n*/\r\n\r\n\r\n// On recupère l'action passée dans l'URL\r\nif (!isset($_POST['action'])) {\r\n    $action = \"afficherAccueilEtu\";\r\n}\r\nif (!isset($_GET['controleur'])) {\r\n    $controleur = \"MainControleur\";\r\n}\r\n\r\n$nomDeClasseControleur = \"App\\Covoiturage\\Controleur\\Controleur\" . ucfirst($controleur);\r\n\r\n$nomDeClasseControleur::$action();\r\n// Appel de la méthode statique $action de ControleurVoiture\r\n\r\n\r\n\r\n
===================================================================
diff --git a/web/ControleurFrontale.php b/web/ControleurFrontale.php
--- a/web/ControleurFrontale.php	
+++ b/web/ControleurFrontale.php	
@@ -1,12 +1,13 @@
 <?php
-/*
+use App\FormatIUT\Controleur\MainControleur as MainControleur;
+
 require_once __DIR__ . '/../src/Lib/Psr4AutoloaderClass.php';
 // initialisation
-$loader = new App\Covoiturage\Lib\Psr4AutoloaderClass();
+$loader = new App\FormatIUT\Lib\Psr4AutoloaderClass();
 $loader->register();
 // enregistrement d'une association "espace de nom" → "dossier"
-$loader->addNamespace('App\Covoiturage', __DIR__ . '/../src');
-*/
+$loader->addNamespace('App\FormatIUT', __DIR__ . '/../src');
+
 
 
 // On recupère l'action passée dans l'URL
@@ -17,10 +18,9 @@
     $controleur = "MainControleur";
 }
 
-$nomDeClasseControleur = "App\Covoiturage\Controleur\Controleur" . ucfirst($controleur);
-
-$nomDeClasseControleur::$action();
-// Appel de la méthode statique $action de ControleurVoiture
+$nomDeClasseControleur = 'App\FormatIUT\Controleur\MainControleur.php';
+MainControleur::afficherAccueilEtu();
+//$nomDeClasseControleur::$action();
 
 
 
