Index: web/ControleurFrontale.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n/*\r\nrequire_once __DIR__ . '/../src/Lib/Psr4AutoloaderClass.php';\r\n// initialisation\r\n$loader = new App\\Covoiturage\\Lib\\Psr4AutoloaderClass();\r\n$loader->register();\r\n// enregistrement d'une association \"espace de nom\" → \"dossier\"\r\n$loader->addNamespace('App\\Covoiturage', __DIR__ . '/../src');\r\n*/\r\n\r\n\r\n// On recupère l'action passée dans l'URL\r\nif (!isset($_POST['action'])) {\r\n    $action = \"afficherAccueilEtu\";\r\n}\r\nif (!isset($_GET['controleur'])) {\r\n    $controleur = \"MainControleur\";\r\n}\r\n\r\n$nomDeClasseControleur = \"App\\Covoiturage\\Controleur\\Controleur\" . ucfirst($controleur);\r\n\r\n$nomDeClasseControleur::$action();\r\n// Appel de la méthode statique $action de ControleurVoiture\r\n\r\n\r\n\r\n
===================================================================
diff --git a/web/ControleurFrontale.php b/web/ControleurFrontale.php
--- a/web/ControleurFrontale.php	
+++ b/web/ControleurFrontale.php	
@@ -1,26 +1,23 @@
 <?php
-/*
+
 require_once __DIR__ . '/../src/Lib/Psr4AutoloaderClass.php';
 // initialisation
-$loader = new App\Covoiturage\Lib\Psr4AutoloaderClass();
+$loader = new App\FormatIUT\Lib\Psr4AutoloaderClass();
 $loader->register();
 // enregistrement d'une association "espace de nom" → "dossier"
-$loader->addNamespace('App\Covoiturage', __DIR__ . '/../src');
-*/
+$loader->addNamespace('App\FormatIUT', __DIR__ . '/../src');
+
 
 
 // On recupère l'action passée dans l'URL
 if (!isset($_POST['action'])) {
-    $action = "afficherAccueilEtu";
+    $action = "afficherAccueilEtu";//définit l'action par défaut donc ce sera afficherIndex
 }
 if (!isset($_GET['controleur'])) {
-    $controleur = "MainControleur";
+    $controleur = "\MainControleur";
 }
 
-$nomDeClasseControleur = "App\Covoiturage\Controleur\Controleur" . ucfirst($controleur);
+$nomDeClasseControleur = "App\FormatIUT\Controleur" . ucfirst($controleur);
 
 $nomDeClasseControleur::$action();
-// Appel de la méthode statique $action de ControleurVoiture
-
-
 
Index: src/Controleur/MainControleur.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\nnamespace App\\FormatIUT\\Controleur;\r\n\r\nclass MainControleur\r\n{\r\n\r\n    public static function afficherAccueilEtu(){\r\n        self::afficherVue(\"vueGenerale.php\",[\"menu\"=>self::getMenu(),\"chemin\"=>\"vueAccueilEtudiant.php\"]);\r\n    }\r\n\r\n    public static function afficherVue(string $cheminVue, array $parametres = []): void\r\n    {\r\n        extract($parametres); // Crée des variables à partir du tableau $parametres\r\n        require __DIR__ . \"/../vue/$cheminVue\"; // Charge la vue\r\n    }\r\n\r\n    public static function getMenu() :array{\r\n        return array();\r\n    }\r\n}
===================================================================
diff --git a/src/Controleur/MainControleur.php b/src/Controleur/MainControleur.php
--- a/src/Controleur/MainControleur.php	
+++ b/src/Controleur/MainControleur.php	
@@ -16,6 +16,12 @@
     }
 
     public static function getMenu() :array{
-        return array();
+        return array(
+            "Menu" =>"lien",
+            "About" =>"lien",
+            "Blogs" =>"lien",
+            "PortFolio" =>"lien",
+            "Contact" =>"lien"
+        );
     }
 }
\ No newline at end of file
